lib/livebook_web/live/hub/new_live.ex

```diff
@@ -158,39 +178,51 @@ defmodule LivebookWeb.Hub.NewLive do
     """
   end
 
-  defp card_item(assigns) do
+  def tab_button(assigns) do
     assigns = assign_new(assigns, :disabled, fn -> false end)
 
     ~H"""
-    <div
-      id={@id}
-      class="flex flex-col cursor-pointer"
-      phx-click={JS.push("select_option", value: %{value: @id})}
-    >
-      <div class={[
-        "flex items-center justify-center p-6 border-2 rounded-t-2xl h-[150px]",
-        card_item_border_class(@id, @selected)
-      ]}>
-        <%= render_slot(@logo) %>
-      </div>
-      <div class={["px-6 py-4 rounded-b-2xl grow", card_item_class(@id, @selected)]}>
-        <p class="text-gray-800 font-semibold cursor-pointer">
-          <%= @title %>
-        </p>
-
-        <p class="mt-2 text-sm text-gray-600">
-          <%= render_slot(@headline) %>
-        </p>
-      </div>
-    </div>
+    <li class="group/toggle w-full">
+      <button
+        type="button"
+        id={@id}
+        aria-haspopup="menu"
+        aria-expanded="false"
+        data-state="closed"
+        class="w-full"
+        phx-click={JS.push("select_option", value: %{value: @id})}
```

Comment:
```suggestion
        phx-click="select_option"
        phx-value={@id}
```

Reply:
It didn't work =/ 

`[error] GenServer #PID<0.7226.0> terminating
** (FunctionClauseError) no function clause matching in LivebookWeb.Hub.NewLive.assign_form/2
    (livebook 0.9.2) lib/livebook_web/live/hub/new_live.ex:308: LivebookWeb.Hub.NewLive.assign_form(#Phoenix.LiveView.Socket<id: "phx-F2JaLS1eF4CCaabB", endpoint: LivebookWeb.Endpoint, view: LivebookWeb.Hub.NewLive, parent_pid: nil, root_pid: #PID<0.7226.0>, router: LivebookWeb.Router, assigns: %{__changed__: %{selected_option: true}, button_label: "Create", confirm_opt_out_ids: MapSet.new([]), confirm_state: nil, current_user: %Livebook.Users.User{id: "zakxm5owubb6kburrm2ao6wbadxvl2oz", name: "Paulo Valim", hex_color: "#6EE7B7"}, enabled?: true, flash: %{}, form: %Phoenix.HTML.Form{source: #Ecto.Changeset<action: nil, changes: %{}, errors: [name: {"can't be blank", [validation: :required]}], data: #Livebook.Teams.Org<>, valid?: false>, impl: Phoenix.HTML.FormData.Ecto.Changeset, id: "org", name: "org", data: %Livebook.Teams.Org{id: nil, emoji: "⭐️", name: nil, teams_key: "Alc7xWF4AJiZ9RCze7YAA3lq8dDPlqFjI2xoisYzzVo", user_code: nil}, hidden: [], params: %{}, errors: [], options: [method: "post"], index: nil, action: nil}, live_action: :new, org: %Livebook.Teams.Org{id: nil, emoji: "⭐️", name: nil, teams_key: "Alc7xWF4AJiZ9RCze7YAA3lq8dDPlqFjI2xoisYzzVo", user_code: nil}, page_title: "Hub - Livebook", request_code_info: "Access the following URL and input the User Code below to confirm to join an Organization.", requested_code: false, saved_hubs: [%Livebook.Hubs.Metadata{id: "personal-hub", name: "My Hub", provider: #Livebook.Hubs.Personal<id: "personal-hub", hub_name: "My Hub", hub_emoji: "🏠", ...>, emoji: "🏠", connected?: false}], selected_option: "", verification_uri: nil}, transport_pid: #PID<0.7219.0>, ...>, "")
    (livebook 0.9.2) lib/livebook_web/live/hub/new_live.ex:227: LivebookWeb.Hub.NewLive.handle_event/3`

Reply:
This should

```suggestion
        phx-click="select_option"
        phx-value-value={@id}
```